_databases/le-superficie-algebriche.md

```diff
@@ -0,0 +1,51 @@
+---
+id: le-superficie-algebriche
+location: "https://superficie.info"
+title: "Le Superficie Algebriche"
+license: "CC BY-SA 4.0" # TODO ENCOURAGED, the license under which the data may be used
+num_datasets: 4 # TODO OPTIONAL, number of collections of different objects; omit if equal to 1
+size: 87654321 # TODO OPTIONAL, the approximate size in bytes
+is_compressed: true # TODO OPTIONAL, whether the size refers to a compressed file
+generation_time: 99999 # TODO OPTIONAL, the approximate total time spent in generating the data in seconds on one CPU
+code_location: "https://github.com/superficie/superficie-algebriche"
+start_date: 1879 # TODO OPTIONAL, approximate year when the project started
+accessible: true # TODO OPTIONAL, data can still be accessed at location above
+completeness: >
+  We include all foo up to order 42
+  and all bar of order up to 5077 # OPTIONAL, a string describing the data's completeness, encouraged if it has the completeness_guarantee badge
+searchable: true
+short_description: "A tool to visually study the geography of (minimal) complex algebraic smooth surfaces."
+contact_email: superficie.algebriche@gmail.com
+authors:
+  - name: Pieter Belmans
+    homepage: "https://pbelmans.ncag.info"
+  - name: Johan Commelin
+    homepage: https://math.commelin.net
+references: # TODO OPTIONAL, include as many as you like
+  - url: "https://other_databaase.com"
+  - doi: 10.0000/0000
+  - arxiv: 9999.54321
+  - mr: 123456
+  - paper: "Emmy Noether and Leonhard Euler.  The Best Paper.  Annals of Math 1 (1879) no 2. pp 1-16."
+  - issn: 12345678
+  - other: "Satellite imagery stored on Mars." # Please avoid; only use if none of the above are appropriate
+badges: # TODO OPTIONAL, keep the ones that apply
+  - collaborative
+  - has_license
+  - downloadable
+  - citable
+  - completeness_guarantee
+  - cas_connection
+  - fm_connection
+  - content_documentation
+  - schema3 # schema description quality: 3/3
+  - has_labels
+  - has_code # code that generates the data is available
+---
+
+This database contains the best work of Noether and Euler, in one corpus.
+
+Here are some examples of the amazing objects within:
+
+- All finite rings
+- All quadratic forms
```

Comment:
```suggestion
This database visualizes the classification of minimal smooth projective complex algebraic surfaces.

For each entry we compute and display several invariants, such as Kodaira dimension, Chern numbers, irregularity, Betti numbers, the Hodge diamond, and more.
```

Reply:
We also include some non-projective ones, to be really precise.

Reply:
Ok, removing "projective":
```suggestion
This database visualizes the classification of minimal smooth complex algebraic surfaces.
For each entry we compute and display several invariants, such as Kodaira dimension, Chern numbers, irregularity, Betti numbers, the Hodge diamond, and more.
```

Reply:
To be even more precise: I should have said _non-algebraic_ (which implies they are not projective, but they are however compact).

So "minimal smooth compact complex surfaces".

Reply:
Of course... I was too fast.

```suggestion
This database visualizes the classification of minimal smooth compact complex algebraic surfaces.
For each entry we compute and display several invariants, such as Kodaira dimension, Chern numbers, irregularity, Betti numbers, the Hodge diamond, and more.
```